cosmology.c

```diff
@@ -43,6 +44,9 @@ void init_cosmology(Cosmology * CP_in)
      * rho_nu/rho_g = 7/8 (T_nu/T_cmb)^4 *N_eff,
      * but we absorbed N_eff into T_nu above. */
     CP->OmegaNu0 = CP->OmegaG * 7. / 8 * pow(TNu0_TCMB0, 4) * 3;
+
+    /*Check whether the dark energy sector is a cosmological constant.*/
+    CP->EvolvingDEOn = (fabs(CP->w0 + 1) < 1e-7) && (fabs(CP->wa) < 1e-7);
```

Comment:
Is this also called the Dark Energy Fluid in CLASS? Perhaps call it that? When Fluid DE is on, OmegaLambda shall be usually set to zero for a 'reasonable' cosmology -- may want to catch that and issue a warning.

I think Class called the parameters just w0_fld and wa_fld -- even if you'd avoid Fluid, DarkEnergy is always more readable.

Reply:
The idea there is you can have a Lambda-like DE and a Fluid like DE component, and they can coexist.

Reply:
Not sure - I was following CAMB and PDG which use definitions more like the ones I did here. class also uses a different definition of wa than I have: w = w0 + wa (1-a0/a), whereas I have w = w0 + wa(1-a). Let me check with a DE cosmologist which one they expect.

Reply:
(this is a CLASS documentation typo)